changelog/fqn_imports_bypass_private_imports_error.dd

```diff
@@ -0,0 +1,20 @@
+The deprecation phase for fully qualified names that bypassed private imports is finished
+
+---
+=========================
+// a.d
+import std.stdio;
+=========================
+// b.d:
+--
+import a;
+
+void main()
+{
+    std.stdio.writefln("foo");         // deprecation before patch, now errors
+}
+==========================
```

Comment:
![image](https://user-images.githubusercontent.com/4370550/42317662-2c74ed96-804d-11e8-8eb6-a847d3d71eb2.png)

Maybe use two separate file sections in the changelog entry?